### ISSUE TITLE
Drop down elements are not getting visual focus except for the element already in selected state in "Working Platform"(Console/WPF/IOS/Android) dropdown beside help button while navigating with in the dropdown in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#384]
### Steps to Reproduce

1. Change the High contrast standards to High Contrast black before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Navigate through working platform (Console/WPF/IOS/Android) dropdown beside "Help" menu item and navigate through the dropdown elements.
### Expected Behavior
Dropdown elements should get visual focus while navigating in the dropdown
### Actual Behavior
Except the element in selected state, rest of the elements in the "Working Platform"(Console/WPF/IOS/Android) dropdown beside help menu item are not getting the visual keyboard focus around them in high contrast black mode.

Note: In high contrast white mode dotted rectangular focus appears on working platform dropdown elements.
### User Impact
Low vision user will not know whether the element got focus or not.
### Screenshots
![20 hcb](https://user-images.githubusercontent.com/22956130/40763812-dfab0f12-64c3-11e8-8688-75d8f72eab50.png)
![20 hcw](https://user-images.githubusercontent.com/22956130/40763813-dfd7633c-64c3-11e8-8957-556ac9d59efc.png)
![20 normal mode](https://user-images.githubusercontent.com/22956130/40763814-e0032080-64c3-11e8-8eae-3dca927685a8.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
